src/PointwiseFunctions/GeneralRelativity/Transform.hpp

```diff
@@ -35,9 +49,31 @@ auto to_different_frame(const tnsr::ii<DataVector, VolumeDim, SrcFrame>& src,
 //@}
 
 //@{
-/// Transforms only the first index to different frame.  Note that
-/// `Jacobian<Dest,Src>` is the same type as `InverseJacobian<Src,Dest>`
-/// and represents \f$ dx^{\rm src}/dx^{\rm dest}\f$.
+/// \ingroup GeneralRelativityGroup
+/// Transforms only the first index to different frame.
+///
+/// Often used for derivatives: When representing derivatives as
+/// tensors, the first index is typically the derivative index.
+/// Numerical derivatives must be computed in the logical frame or
+/// sometimes the grid frame (independent of the frame of the tensor
+/// being differentiated), and then that derivative index must later
+/// be transformed into the same frame as the other indices of the
+/// tensor.
```

Comment:
Isn't this what our `partial_derivatives` function does?

Reply:
Yes, but our partial_derivatives function doesn't let you put the result into a `Tensor` inside an already-allocated `Variables`, or allow the `Variables` holding the result to be the same `Variables` as the one containing the `Tensor` to be differentiated, or allow the subset of `Variables` tags that are to be differentiated to occur anywhere other than the beginning of the tag list of the `Variables`.

Reply:
OK, that's a reasonable use for this.